src/Trixi.jl

```diff
@@ -139,7 +142,8 @@ export initial_condition_constant,
        initial_condition_orszag_tang,
        initial_condition_rotor,
        initial_condition_shock_bubble,
-       initial_condition_taylor_green_vortex
+       initial_condition_taylor_green_vortex,
+       initial_condition_well_balancedness
```

Comment:
This should be possible although it will make it such that there are two initial condition functions in `elixir_shallowwater_well_balanced.jl` one that can be used by Trixi to compute the errors (which are needed for the reference values) and another initial condition with my hacky way of setting the discontinuous bottom topography.

Reply:
Yeah, that's okay.

Reply:
Done